packages/app/src/app/overmind/effects/live/index.ts

```diff
@@ -495,6 +495,18 @@ class Live {
     });
   }
 
+  reset() {
+    this.clients.clear();
+
+    if (this.awaitSendTimer) {
+      clearTimeout(this.awaitSendTimer);
+    }
+
+    if (this.awaitSend) {
+      this.awaitSend = null;
+    }
```

Comment:
```suggestion
    this.awaitSend = null;
```

No need to check, just set it to null :-)